src/java.base/share/native/libjava/System.c

```diff
@@ -297,6 +303,7 @@ static void cpchars(jchar *dst, char *src, int n)
     }
 }
 
+#if 0 /* Exclude so it doesn't conflict with the OpenJ9 native. */
```

Comment:
I expect we need to move this before `cpchars()` else warnings will ensue that that function is unused.

Reply:
Moved it.